### ISSUE TITLE
Couldn't find the terminal (v.0.5)

### ISSUE THREAD

**Issue by DrPerry97** [malebuffy/Kobra-Max-Cobra-Kai-UI-and-Firmware#5]
My updates are succefull and it runs, but I couldn't find the terminal. 
And the thumbnail.py doesn't appear in Cura. Other scripts are in the 
same place and working. I am confused. 
Cura 5.3.1

Any suggestions for me ?

**Comment by malebuffy** [comment]
Hi,

for the first problem, did you also update the screen with the DWIN_SET folder? Using the Screen SD Slot?

Also make sure the file should be called CreateKobraThumbnail.py and a restart of Cura is needed.

**Comment by Woor** [comment]
I'm running into the same issue. Installed o.5 firmware (in main slot), DWIN_Set in screen slot after that. I can see Kobra Kai logo, previews of my prints and all, but there's no Terminal app visible anywhere?

What's worse is that apparently the terminal makes Pronterface unhappy, it connects, but then immediately disconnects again, telling me there's other access going on, I assume that is the Terminal running.

Is there a way to disable the Terminal? If it interferes with Pronterface and the likes, I don't think it's such a good idea to have it in there.

**Comment by malebuffy** [comment]
The idea of the terminal app was that some boards like mine have usb not working. So that's why i created it. I have however one board where usb is working. I will troubleshoot.

If you are interested only in the preview,  you could use a prior release.

**Comment by Woor** [comment]
Thank you for the answer.
The serial issue seems to be on the computer's end, not related to your firmware. The Terminal icon not being visible seems to persist though. Using 0.4 version of DWIN_SET shows it correctly again.

**Comment by NeuralDEEPnet** [comment]
Yeah please include tem in 0.5

**Comment by malebuffy** [comment]
Hi, I  don't really understand. Do you mean that the DWIN_SET UI firmware doesn't include the Terminal menu?

**Comment by NeuralDEEPnet** [comment]
Yes 0.5 version dotn have terminal. In gen I went back to stock firmware.
(gcode didnt help) Yours have major probs with extruder. Please make one
with good extruder performance. Otherwise good UX ui .
_____________________________________

Tomas Jankauskas
Founder, Neuraldeep.net
***@***.***
+46702928774

Creative #3DATA Solutions
_____________________________________

Unleash Your Ideas into Reality

🌌  <http://www.neuraldeep.net>//////////////////Book a meeting with me
///////////// <https://koalendar.com/e/NeuralDEEPnet> 🌌
<http://www.neuraldeep.net>
